lib/modules/platform/azure/index.spec.ts

```diff
@@ -1985,4 +1985,117 @@ describe('modules/platform/azure/index', () => {
       expect(res).toBeNull();
     });
   });
+
+  describe('pr cache check', () => {
+    it('check PR create and update cache', async () => {
+      await initRepo({ repository: 'some/repo' });
+      azureApi.gitApi.mockImplementationOnce(
+        () =>
+          ({
+            getPullRequests: jest
+              .fn()
+              .mockReturnValue([])
+              .mockReturnValueOnce([
+                {
+                  pullRequestId: 1,
+                  sourceRefName: 'refs/heads/branch-a',
+                  targetRefName: 'refs/heads/branch-b',
+                  title: 'branch a pr',
+                  status: PullRequestStatus.Active,
+                },
+              ]),
+            createPullRequest: jest.fn(() => ({
+              pullRequestId: 1,
+            })),
+            createPullRequestLabel: jest.fn(() => ({})),
+          }) as any,
+      );
+
+      await azure.createPr({
+        sourceBranch: 'branch-a',
+        targetBranch: 'branch-c',
+        prTitle: 'branch a pr',
+        prBody: 'Hello world',
+        labels: ['renovate'],
+      });
+
+      let prList = await azure.getPrList();
+
+      expect(prList).toHaveLength(1);
+      expect(prList[0].number).toBe(1);
+
+      azureApi.gitApi.mockImplementationOnce(
+        () =>
+          ({
+            getPullRequests: jest
+              .fn()
+              .mockReturnValue([])
+              .mockReturnValueOnce([
+                {
+                  pullRequestId: 1,
+                  sourceRefName: 'refs/heads/branch-a',
+                  targetRefName: 'refs/heads/branch-b',
+                  title: 'branch a pr',
+                  status: PullRequestStatus.Active,
+                },
+              ]),
+            createPullRequest: jest.fn(() => ({
+              pullRequestId: 1,
+              sourceRefName: 'refs/heads/branch-a',
+              targetRefName: 'refs/heads/branch-b',
+              title: 'branch a pr',
+              status: PullRequestStatus.Active,
+            })),
+            createPullRequestLabel: jest.fn(() => ({})),
+          }) as any,
+      );
+      // create again same pr
```

Comment:
This is an invalid test. Azure does not permit the creation of the same PR like this, and even if branch name and title were the same, then the PR ID would not be the same